src/components/paint-editor/paint-editor.jsx

```diff
@@ -257,6 +256,41 @@ class PaintEditorComponent extends React.Component {
                             svgId={this.props.svgId}
                             onUpdateSvg={this.props.onUpdateSvg}
                         />
+                        {/* Zoom controls */}
+                        <InputGroup className={styles.zoomControls}>
+                            <ButtonGroup>
+                                <Button
+                                    className={styles.buttonGroupButton}
+                                    onClick={this.props.onZoomIn}
+                                >
+                                    <img
+                                        alt="Zoom In Icon"
```

Comment:
Sorry I didn't catch this in earlier reviews. Our alt text shouldn't include 'icon'. I think the 'button' context is automatically added by screen readers so it can just say 'Zoom in'